openapi-docs/docs/MiscApi.md

```diff
@@ -0,0 +1,224 @@
+# AmbianicFunctionsCollection.MiscApi
```

Comment:
@vickywane why are there two sets of files with overlapping API definitions `DeaultApi` and `MiscApi` ?

Reply:
I have deleted the duplicate. It came from a name change in the swagger file.